ui/dashboards/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@perses-dev/dashboards",
+  "version": "0.1.0",
+  "description": "The dashboards feature in Perses",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "clean": "rimraf dist/",
+    "build": "tsc",
+    "test": "echo 'no test to run' && exit 0",
+    "lint": "eslint src --ext .ts,.tsx",
+    "lint:fix": "eslint --fix src --ext .ts,.tsx"
+  },
+  "dependencies": {
+    "@mui/material": "^5.4.0",
+    "@perses-dev/components": "^0.1.0",
+    "@perses-dev/core": "^0.1.0",
+    "@perses-dev/plugin-system": "^0.1.0",
```

Comment:
ah interesting to see that this package doesn't depend on `prometheus-plugin`

Reply:
Yeah, it can just depend on the `plugin-system` package and not know anything about what plugin is being used to do the querying for data. So in the (far) future, we could render data from things like Graphite, etc.